internal/tiltfile/starkit/fixture.go

```diff
@@ -12,25 +12,37 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.starlark.net/starlark"
+
+	"github.com/windmilleng/tilt/internal/testutils/tempdir"
 )
 
 // A fixture for test setup/teardown
 type Fixture struct {
-	tb         testing.TB
-	extensions []Extension
-	path       string
-	fs         map[string]string
-	out        *bytes.Buffer
-	useRealFS  bool // Use a real filesystem
+	tb               testing.TB
+	extensions       []Extension
+	path             string
+	temp             *tempdir.TempDirFixture
+	fs               map[string]string
+	out              *bytes.Buffer
+	useRealFS        bool // Use a real filesystem
+	loadInterceptors []LoadInterceptor
+	oldDir           string
 }
 
 func NewFixture(tb testing.TB, extensions ...Extension) *Fixture {
+	temp := tempdir.NewTempDirFixture(tb)
+	oldDir, err := os.Getwd()
```

Comment:
can we make changing the working directory an option on TempDirFixture?

Reply:
Yeah! And I would also like to socialize that out to the other tests where we do similar things.